src/content/en/updates/2016/10/navigator.share.md

```diff
@@ -5,7 +5,7 @@ description: Sharing is caring. Chrome is running an Origin Trial to enable nati
 {# wf_published_on: 2016-10-19 #}
 {# wf_updated_on: 2016-10-19 #}
 
-# Introducing the Web Share API {: .page-title}
+# Introducing the Web Share API {: .page-title }
```

Comment:
Though, I did just change the test for this, so it won't fail any more. The other updates are still needed though sorry.